ExampleApp/TidepoolKit Example/LoginView.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  LoginView.swift
+//  TidepoolKit Example
+//
+//  Created by Pete Schwamb on 4/13/23.
+//  Copyright © 2023 Tidepool Project. All rights reserved.
+//
+
+import SwiftUI
+import TidepoolKit
+
+@MainActor
```

Comment:
`@MainActor` here should not be needed

Reply:
The `body` method of SwiftUI views is automatically annotated with `@MainActor`, but if I want to ensure all other methods/accesses happen on the main actor, then I need to do this, or annotate the individual methods, correct? In this specific case, I'm talking about things like the login/logout button actions. Let me know if there is a preferred style for this.